tests/test_mediastoredata/test_mediastoredata.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import unicode_literals
+
+import boto3
+import pytest
+import sure  # noqa
+from botocore.exceptions import ClientError
+
+from moto import mock_mediastoredata
+
+region = "eu-west-1"
+
+
+@mock_mediastoredata
+def test_put_object():
+    client = boto3.client("mediastore-data", region_name=region)
+    response = client.put_object(Body="011001", Path="foo")
+    response["ResponseMetadata"]["HTTPStatusCode"].should.equal(200)
+    response["StorageClass"].should.equal('TEMPORAL')
+    # TODO:  Why Only ETag is available on Items
```

Comment:
'Path' is never returned for the list_items call. See the following example:
https://awscli.amazonaws.com/v2/documentation/api/latest/reference/mediastore-data/list-items.html#examples

From that list, only 'ETag' is currently returned by Moto. If you add type/content_length/name attributes to the response, they should be returned as well.